Directory.Packages.props

```diff
@@ -14,7 +14,7 @@
 		<PackageVersion  Include="RichardSzalay.MockHttp" Version="6.0.0" />
 		<PackageVersion  Include="SimpleLogInterface" Version="3.0.1" />
 		<PackageVersion  Include="System.Collections.Immutable" Version="6.0.0" />
-		<PackageVersion  Include="System.IdentityModel.Tokens.Jwt" Version="6.17.0" />
-		<PackageVersion  Include="System.Text.Json" Version="6.0.4" />
+		<PackageVersion  Include="System.IdentityModel.Tokens.Jwt" Version="6.36.0" />
+		<PackageVersion  Include="System.Text.Json" Version="8.0.4" />
```

Comment:
`System.IdentityModel.Tokens.Jwt` is a legacy library that's been replaced by `Microsoft.IdentityModel.JsonWebTokens`.

This one still always uses the vendored in-assembly Newtonsoft; however the newer one has switched to using System.Text.Json for net461+, hence the dependency.